src/data/polynomial/laurent.lean

```diff
@@ -294,6 +294,15 @@ left_inverse_trunc_to_laurent.injective
   f.to_laurent = g.to_laurent ↔ f = g :=
 ⟨λ h, polynomial.to_laurent_injective h, congr_arg _⟩
 
+instance : module R[X] R[T;T⁻¹] :=
+{ smul      := λ f l, f.to_laurent * l,
+  one_smul  := λ f,     by simp only [map_one, one_mul],
+  mul_smul  := λ f g l, by simp only [mul_assoc, _root_.map_mul],
+  smul_add  := λ f x y, by simp only [mul_add],
+  smul_zero := λ f,     by simp only [mul_zero],
+  add_smul  := λ f g x, by simp only [add_mul, _root_.map_add],
+  zero_smul := λ f,     by simp only [_root_.map_zero, zero_mul] }
```

Comment:
I believe this should golf to `module.comp_hom to_laurent` or something along those lines

Reply:
I'm always amazed at how some of the "trivial" facts also admit "trivial proofs"!